hazelcast/src/main/java/com/hazelcast/instance/DefaultAddressPicker.java

```diff
@@ -187,6 +170,33 @@ private AddressDefinition pickAddress(NetworkConfig networkConfig) throws Unknow
         return addressDef;
     }
 
+    private AddressDefinition pickInterfaceAddress(NetworkConfig networkConfig) throws UnknownHostException, SocketException {
```

Comment:
I just wanted to give a hint which changes are no-brainer (like the one before, which is done by IntelliJ) or are more critical due to manual refactoring (like this one).